shpkpr/exceptions.py

```diff
@@ -36,6 +37,7 @@ def rewrap(exceptions_to_catch, exception_to_rewrap_with=ShpkprException):
     before re-raising with the original stack trace.
     """
     def real_decorator(function):
+        @functools.wraps(function)
```

Comment:
We really should have been doing this all along, it ensures the decorator is a good citizen and propagates the wrapped function's docstring and other metadata to the caller.

I noticed this when sphinx initially refused to document the wrapped template filters, which is why it's in this PR.